src/builder.rs

```diff
@@ -405,7 +418,7 @@ impl SessionBuilder {
             .arg("-f")
             .arg("-N")
             .arg("-o")
-            .arg("ControlPersist=yes")
+            .arg(self.control_persist.as_option().deref())
```

Comment:
Without the `deref()` it complains with `the trait AsRef<OsStr> is not implemented for Cow<'_, str>`

Reply:
Yeah, this is ok